edenai_apis/apis/picsart/picsart_api.py

```
@@ -85,6 +87,63 @@ def image__background_removal(
+
+        files = {}
+        image_file = None
+
+        if file_url:
+            provider_params["image_url"] = file_url
+        elif file:
+            async with aiofiles.open(file, "rb") as f:
+                image_file = await f.read()
+                files = {"image": image_file}
+        else:
+            raise ProviderException("No file or file_url provided")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)